.github/workflows/build.yaml

```diff
@@ -354,7 +360,8 @@ jobs:
   release:
     needs: [build, sign-and-package-windows, sign-and-package-mac]
     runs-on: ubuntu-latest
-    if: github.ref_type == 'tag' && startsWith(github.ref_name, 'Second_Life_')
+      # Build with a tag like "Second_Life#abcdef0" to generate a release page (used for builds we are planning to deploy).
+    if: github.ref_type == 'tag' && startsWith(github.ref_name, 'Second_Life')
```

Comment:
[Caladbolg is advocating](https://lindenlab.slack.com/archives/C02R43FJP/p1714514147094849?thread_ts=1714500309.255049&cid=C02R43FJP) triggering release builds with a tag that includes the branch name -- not to specify the branch name to the build, but for clarity in navigating.

Maybe something like: `branch/<branchname>#discriminator` ?

`tags:` at line 8 would have to change too.